.pullapprove.yaml

```diff
@@ -0,0 +1,34 @@
+version: 2
+
+requirements:
+  signed_off_by:
+    required: true
+
+always_pending:
+  title_regex: '(WIP|wip)'
+  explanation: 'Work in progress...'
+
+group_defaults:
+  approve_by_comment:
+    enabled: true
+    approve_regex: '^(Approved|LGTM|:shipit:|:\+1:|Engage)'
```

Comment:
I used https://github.com/pi-hole/pi-hole/blob/master/.pullapprove.yml#L13 as template. Even when nobody uses "Engage", I would never vote for removing Star Trek eastereggs :)